mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/syncsavingsaccounttransaction/SyncSavingsAccountTransactionFragment.java

```diff
@@ -0,0 +1,233 @@
+        SyncSavingsAccountTransactionMvpView, SwipeRefreshLayout.OnRefreshListener,
+        DialogInterface.OnClickListener {
+
+
+    public final String LOG_TAG = getClass().getSimpleName();
+
+    @BindView(R.id.rv_sync_payload)
+    RecyclerView rv_loan_repayment;
+
+    @BindView(R.id.swipe_container)
+    SwipeRefreshLayout swipeRefreshLayout;
+
+    @BindView(R.id.noPayloadText)
+    TextView mNoPayloadText;
+
+    @BindView(R.id.noPayloadIcon)
+    ImageView mNoPayloadIcon;
+
+    @BindView(R.id.ll_error)
+    LinearLayout ll_error;
+
+    @Inject
+    SyncSavingsAccountTransactionPresenter mSyncSavingsAccountTransactionPresenter;
+
+    @Inject
+    SyncSavingsAccountTransactionAdapter mSyncSavingsAccountTransactionAdapter;
+
+    private View rootView;
+
+
+    public static SyncSavingsAccountTransactionFragment newInstance() {
+        Bundle arguments = new Bundle();
+        SyncSavingsAccountTransactionFragment sync = new SyncSavingsAccountTransactionFragment();
+        sync.setArguments(arguments);
+        return sync;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((MifosBaseActivity) getActivity()).getActivityComponent().inject(this);
+        setHasOptionsMenu(true);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        rootView = inflater.inflate(R.layout.fragment_syncpayload, container, false);
+
+        ButterKnife.bind(this, rootView);
+        mSyncSavingsAccountTransactionPresenter.attachView(this);
+
+        LinearLayoutManager mLayoutManager = new LinearLayoutManager(getActivity());
+        mLayoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+        rv_loan_repayment.setLayoutManager(mLayoutManager);
+        rv_loan_repayment.setHasFixedSize(true);
+        rv_loan_repayment.setAdapter(mSyncSavingsAccountTransactionAdapter);
+        swipeRefreshLayout.setColorSchemeColors(getActivity()
+                .getResources().getIntArray(R.array.swipeRefreshColors));
+        swipeRefreshLayout.setOnRefreshListener(this);
+
+        //Loading LoanRepayment Transactions  and PaymentTypeOptions From Database
+        mSyncSavingsAccountTransactionPresenter.loadDatabaseSavingsAccountTransactions();
+        mSyncSavingsAccountTransactionPresenter.loadPaymentTypeOption();
+
+        return rootView;
+    }
+
+    /**
+     * Loading All SavingsAccount Transactions from Database On SwipeRefresh
+     */
+    @Override
+    public void onRefresh() {
+        //Loading LoanRepayment Transactions and PaymentTypeOptions From Database
+        mSyncSavingsAccountTransactionPresenter.loadDatabaseSavingsAccountTransactions();
+        mSyncSavingsAccountTransactionPresenter.loadPaymentTypeOption();
+
+        if (swipeRefreshLayout.isRefreshing())
```

Comment:
Problem here!

write the statement to be executed inside the if block on the same line if you are not using braces.
This way you prevent people and yourself from adding any code below it that might end up being confused with the if block